server/utils/reportUtils.ts

```diff
@@ -1,17 +1,28 @@
 export const reportNames = {
-  'daily-metrics': 'Daily Metrics',
-  'placement-metrics': 'Placement Metrics',
-  'referrals-by-ap-type': 'Referrals by AP Type',
-  'referrals-by-tier': 'Referrals by Tier',
-  applications: 'Combined applications and placement requests',
-  referrals: 'Applications',
-  'lost-beds': 'Lost Beds',
-  'placement-applications': 'Placement Requests',
-  'placement-matching-outcomes': 'Placement Matching Outcomes',
+  referrals: {
+    text: 'Raw Applications',
+    hint: 'A raw data extract for applications submitted within the month. Includes data up to the point of assessment completion.',
+  },
+  'placement-applications': {
+    text: 'Raw requests for placement',
+    hint: 'A raw data extract for request for placements created within the month. Includes application data, but does not include matching or booking data.',
+  },
+  applications: { text: 'Raw combined applications and placement requests', hint: '' },
+  'placement-matching-outcomes': {
+    text: 'Raw data for Placement matching outcomes',
+    hint: 'A raw data extract to help identify placement matching outcomes. This downloads Match requests based on the Expected Arrival Date.',
+  },
+  'lost-beds': {
+    text: 'Lost beds',
+    hint: 'A report on all lost beds for that month and how long they were unavailable for.',
+  },
+  'daily-metrics': { text: 'Daily metrics', hint: 'Counts of key actions across the service grouped by day.' },
 } as const
 
 export type ReportType = (keyof typeof reportNames)[number]
 
-export const reportOptions = Object.entries(reportNames)
-  .sort((a, b) => (a[1] > b[1] ? 1 : -1))
-  .map(e => ({ value: e[0], text: e[1] }))
+export const reportOptions = Object.entries(reportNames).map(e => ({
```

Comment:
Might just be Monday morning but I struggled to grok this.  
I think it would be clearer if `reportNames` was renamed to (something like) `reportInputLabels`. 
In the `.map` here we could deconstruct the parameters and name them:  
```
.map(([reportName, reportLabel]),  => ({
  value: reportName,
  text: reportLabel.text,
  hint: { text: reportLabel.hint })) 
  ```